Tests/UnitTests/Core/Geometry/GenericCuboidVolumeBoundsTests.cpp

```diff
@@ -178,6 +179,27 @@ BOOST_AUTO_TEST_CASE(bounding_box_creation) {
   BOOST_CHECK_EQUAL(bb.entity(), nullptr);
   CHECK_CLOSE_ABS(bb.max(), Vector3(1.00976, 2.26918, 1.11988), tol);
   CHECK_CLOSE_ABS(bb.min(), Vector3(-0.871397, 0, -0.0867708), tol);
+
+  // Check recreation from bound values
+  const auto boundValues = gcvb.values();
+  BOOST_CHECK(boundValues.size() == 24u);
+
+  std::array<ActsScalar, GenericCuboidVolumeBounds::BoundValues::eSize>
```

Comment:
Can you use the to_array function here now that it is in Helpers.hpp?

Reply:
done.